usr/share/job/adapter.slurm.erb

```diff
@@ -0,0 +1,11 @@
+# ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+#  >>>> FLIGHT JOB ADAPTER
+# ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+
+#=====================
+#  Results directory
+#---------------------
+# By convention, job's submitted with `flight-job` are expected to save their
+# results to RESULTS_DIR.  `flight-job` provides easy access to any files
+# saved there.
+RESULTS_DIR="$(pwd)/${SLURM_JOB_NAME}-outputs/$SLURM_JOB_ID"
```

Comment:
This isn't the correct path for array jobs.  It was previously using `RESULTS_DIR="$(pwd)/${SLURM_JOB_NAME}-outputs/$SLURM_ARRAY_JOB_ID"`

Perhaps adding a tag to the array template and branching on that here would work?

Reply:
Using `tags` might be good long term solution, but the quick fix is to toggle on the env var:

```
if [ -n "$SLURM_ARRAY_JOB_ID" ]; then
  RESULTS_DIR="$(pwd)/${SLURM_JOB_NAME}-outputs/$SLURM_ARRAY_JOB_ID"
else
  RESULTS_DIR="$(pwd)/${SLURM_JOB_NAME}-outputs/$SLURM_JOB_ID"
fi
```

4fbe7c15f8a32ab81a5336d25c9ebdba5780d152